pkg/pillar/cmd/diag/diag.go

```diff
@@ -627,6 +627,8 @@ func printOutput(ctx *diagContext) {
 
 	numMgmtPorts := len(types.GetMgmtPortsAny(*ctx.DeviceNetworkStatus, 0))
 	fmt.Fprintf(outfile, "INFO: Have %d total ports. %d ports should be connected to EV controller\n", numPorts, numMgmtPorts)
+	ctxWork, cancel := zedcloud.GetContextForAllIntfFunctions(ctx.zedcloudCtx)
```

Comment:
Does diag also have a 300 second watchdog using a .touch file?

Reply:
I remove the logic from diag. Seems it is not needed there as you mentioned.